ocean-client/src/utils/helpers.ts

```diff
@@ -9,32 +10,34 @@ export function delay(ms: number) {
 }
 
 
-export function nextCollateralValue(vault: LoanVaultActive): number {
-    let nextCollateral = 0
+export function nextCollateralValue(vault: LoanVaultActive): BigNumber {
+    let nextCollateral = new BigNumber(0)
     vault.collateralAmounts.forEach(collateral => {
         if (collateral.symbol == "DUSD") {
-            nextCollateral += Number(collateral.amount) * 0.99 //no oracle price for DUSD, fixed 0.99
+            nextCollateral= nextCollateral.plus(new BigNumber(collateral.amount).multipliedBy(0.99)) //no oracle price for DUSD, fixed 0.99
```

Comment:
because i just stumbled over it again. That "fixed" price for DUSD because there is no oracle for it.
Is it the same like on-chain? Because somehow the chain needs to calculate it too

Reply:
yes, DUSD "oracle" is hardcoded to 1 on chain. no idea why ocean is not providing it as part of the data.